### ISSUE TITLE
Front End Highlighting: Insufficient Color Contrast not being found

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/accessibility-checker#161]
Insufficient Color Contrast is not being found most likely because the CSS is being tagged onto the code snippet. Is there are way we can remove the CSS before the script tries to find the element?

<img width="397" alt="Screen Shot 2023-06-06 at 11 36 55 AM" src="https://github.com/equalizedigital/accessibility-checker/assets/2895788/8b61ad84-6155-4e8b-a5c0-5977ce4d2bb9">

**Comment by boonedev** [comment]
@SteveJonesDev could you send over the original html code for the form that's not being found?

I'm testing with these two html blocks:
<img width="527" alt="Screenshot 2023-06-06 at 11 42 57 AM" src="https://github.com/equalizedigital/accessibility-checker/assets/695220/1f4be18e-a1fe-4bd4-a855-24fe71e265a7">

It catches the first, but not the second.

**Comment by boonedev** [comment]
Short answer: 
All fixed. And Show Code now only shows the html snippet (so no css text)

Long Answer:
It looks like css text isn't the issue b/c findElement() uses firstElementChild which strips out the appended css (NodeText). I think the problem is that I wasn't cleaning up the element-id attribute that I added to the problem element. So:

1) When you first open the panel it runs findElement just fine, then adds the element-id on the element that has an issue
2) When you close the panel, it cleans up the tooltip/buttons- but leaves the element-id attr.
3) Then, if you open the panel again, findElement fails b/c the html no longer matches

I added the cleanup and I think things are working well now.

**Comment by SteveJonesDev** [comment]
@boonedev, there is now an issue with the issues not closing when the control panel is closed. Screen recording: https://www.dropbox.com/s/84xpqg98phjhg0t/Screen%20Recording%202023-06-07%20at%2012.16.54%20PM.mov?dl=0

**Comment by boonedev** [comment]
@SteveJonesDev ... moving too fast. Will get it corrected. Thanks!

**Comment by boonedev** [comment]
@SteveJonesDev commit a4bcad0e641c679da5e3fbca27906bb0bf6845e2 should fix - could you let me know if not?

**Comment by SteveJonesDev** [comment]
This looks good. Closing out the issue as the bug with the show code button is addressed in another open issue.
